src/components/Login.tsx

```diff
@@ -174,7 +174,16 @@ export default function Login({ refresh }: Props) {
               })}
               onClick={() => handleChangeLanguage('tr')}
             >
-              Türkçe 🇹🇷
+              Türkçe 🇹🇷 -{' '}
+            </span>
+            <span
+              className={cx({
+                ['selectedLanguage']: currentLanguage === 'es',
+                ['language']: currentLanguage !== 'es',
+              })}
+              onClick={() => handleChangeLanguage('es')}
+            >
+              Español 🇲🇽
```

Comment:
I know this is a difficult question, but maybe we should use the Spanish flag instead? 
Even though then we should change the Portuguese one to the Portuguese flag 🤔